ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/eip4844/helpers/MiscHelpersEip4844.java

```diff
@@ -75,10 +76,12 @@ private void validateBlobSidecar(
     checkArgument(
         kzgCommitments.size() == blobsSidecar.getBlobs().size(),
         "Number of kzgCommitments should match number of blobs");
-    kzg.verifyAggregateKzgProof(
-        blobsSidecar.getBlobs().stream().map(Blob::getBytes).collect(Collectors.toList()),
-        kzgCommitments,
-        blobsSidecar.getKZGAggregatedProof());
+    final boolean isValidProof =
+        kzg.verifyAggregateKzgProof(
+            blobsSidecar.getBlobs().stream().map(Blob::getBytes).collect(Collectors.toList()),
+            kzgCommitments,
+            blobsSidecar.getKZGAggregatedProof());
+    checkState(isValidProof, "The proof for the blobs and the commitments is not valid");
```

Comment:
I might keep the message simple and just use the method name, so something like "Invalid aggregate kzg proof"

Reply:
I like "Invalid aggregate kzg proof"